### PR TITLE
util: fix install.sh mn_deps step on Debian 11

### PR DESCRIPTION
**Pull request by yoursunny**

### CODE REVIEW

**Comment by lantz** [comment]
For python 2, the ubuntu 20 package is still called pep8

File: examples/multiping.py [hunk 1]
```diff
@@ -17,9 +17,9 @@
 from mininet.log import info, setLogLevel
 
 
-def chunks( l, n ):
-    "Divide list l into chunks of size n - thanks Stackoverflow"
-    return [ l[ i: i + n ] for i in range( 0, len( l ), n ) ]
+def chunks( lst, n ):
```
Comment (yoursunny) [inline examples/multiping.py:20]: This is to fix pycodestyle 2.6.0 (Debian 11) error:

> mininet/examples/multiping.py:20:13: E741 ambiguous variable name 'l'

File: mininet/cli.py [hunk 2]
```diff
@@ -455,13 +455,6 @@ def waitForNode( self, node ):
         while True:
             try:
                 bothPoller.poll()
-                # XXX BL: this doesn't quite do what we want.
```
Comment (yoursunny) [inline mininet/cli.py:458]: This is to fix pycodestyle 2.6.0 (Debian 11) error:

> mininet/cli.py:459: [R1727(condition-evals-to-constant), CLI.waitForNode] Boolean condition 'False and self.inputFile' will always evaluate to 'False'

File: examples/miniedit.py [hunk 3]
```diff
@@ -1429,8 +1429,11 @@ def convertJsonUnicode(self, text):
             return {self.convertJsonUnicode(key): self.convertJsonUnicode(value) for key, value in text.items()}
         if isinstance(text, list):
             return [self.convertJsonUnicode(element) for element in text]
-        if isinstance(text, unicode):  # pylint: disable=undefined-variable
-            return text.encode('utf-8')
+        try:
```
Comment (yoursunny) [inline examples/miniedit.py:1432]: This is to fix pyflakes3 2.1.1 (Ubuntu 20) error:

> examples/miniedit.py:1432: undefined name 'unicode'

File: util/install.sh [hunk 4]
```diff
@@ -174,15 +174,15 @@ function mn_deps {
...
     else  # Debian/Ubuntu
         pf=pyflakes
-        # Starting around 20.04, installing pyflakes instead of pyflakes3
+        # Starting Ubuntu 18.04, installing pyflakes instead of pyflakes3
         # causes Python 2 to be installed, which is exactly NOT what we want.
-        if [ `expr $RELEASE '>=' 20.04` = "1" ]; then
-                pf=pyflakes3
+        if ( [[ $DIST == Ubuntu ]] && [[ `expr $RELEASE '>=' 18.04` == 1 ]] ) ||
+           ( [[ $DIST == Debian ]] && [[ $RELEASE -ge 10 ]] ); then
+            pf=pyflakes3
```
Comment (yoursunny) [inline util/install.sh:181]: > For python 2, the ubuntu 20 package is still called pep8, so this breaks the python 2 code check.
> It also appears that pep8 has been renamed pycodestyle for recent Ubuntu (and I presume debian) releases.

That's not the case.

* `pep8` package is an alias of `pycodestyle` that is for Python 3. It is available in Ubuntu 18, Ubuntu 20, Debian 11.
* `python-pep8` package is for Python 2. It is only available in Ubuntu 18.

The existing script always installs `pep8` regardless of Python version.
I'm changing it to `pycodestyle` to reduce indirection.

**Review by yoursunny — COMMENTED** [review]
I have tested these changes with `make codecheck` on:

* Ubuntu 20 + Python 2 (manually install `python-is-python2`)
* Ubuntu 20 + Python 3 (manually install `python-is-python3`)
* Debian 11 + Python 3 (manually install `python-is-python3`)

**Comment by yoursunny** [comment]
@lantz is there any other change needed before this can be merged?

**Comment by yoursunny** [comment]
Replaced by #1118.

**Pull request closed without merging**